src/components/goalDetail/GoalHeader.tsx

```
@@ -1,152 +1,193 @@
-import { useGoals, useUpdateGoal, useDeleteGoal } from '@/hooks/useGoals';
-import GoalProgress from './GoalProgress';
 import { Goal } from '@/types/goals';
+import { useGoals, useUpdateGoal, useDeleteGoal } from '@/hooks/useGoals';
+import goalColors from '@/presets/goalColors';
 
-type Props = {
-  doneItems: number;
-  todoItems: number;
+interface Props {
   id: string;
   setGoalAvailable: (value: boolean) => void;
```

ESLint

[error] 'value' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)